roles/senior_software_engineer_2.md

```diff
@@ -24,8 +26,6 @@ We grow a team of polyglot programmers, which you might already consider yoursel
 
 ## What experience are we looking for?
 
-While we will look for you to have experience in these things, if you don’t have one of these don’t let that stop you applying.
```

Comment:
I'm in favour of keeping this paragraph, or something like it. Less represented groups are proportionately less likely to apply if they don't have 100% of the "requirements" for a job.

Reply:
It's a mistake, will add back.